tests/integration/components/build-header-test.js

```diff
@@ -219,4 +219,40 @@ module('Integration | Component | build header', function (hooks) {
     await render(hbs`{{build-header item=job}}`);
     assert.dom('.detail-job-arch').exists('does display CPU architecture');
   });
+
+  test('show prioritize heading', async function (assert) {
```

Comment:
Could you please make test titles a little bit more descriptive, e.g. "it shows prioritize heading when build is of high priority"?

Reply:
It got missed, sorry now fixed.